resources/metadata/whitesource.yaml

```diff
@@ -398,109 +398,10 @@ spec:
     resources:
       - name: buildDescriptor
         type: stash
-        conditions:
-          - conditionRef: strings-equal
-            params:
-              - name: scanType
-                value: golang
```

Comment:
I'd like to understand why those conditions were there in the first place. Probably left-over from early days of the step when it could only handle go projects?

Reply:
Several reasons:
* Upon a closer look, these conditions enable the same 2 stashes for every supported scanType, these two stashes should be enabled unconditionally. The third stash, "checkmarx", was not enabled for every scanType, but it is now also enabled unconditionally. I have not investigated whether it is truly needed.
* The problem with the conditions was that "scanType" is supposed to be derived from the parameter "buildTool", if not supplied, as also in the Groovy version. But at the point in time where the stashes are evaluated, scanType has not yet been filled from buildTool, so no stashes were effective at all.
* Also worth noting is that "scanType" can have any value, and that will trigger the Unified Agent scanning. Thus, there should not be any conditions bound to the value of scanType which would cause inconsistency with the allowed values of this parameter.